app/Http/Controllers/SekolahController.php

```
@@ -0,0 +1,219 @@
+        return view('sekolah.guru', compact('gurus', 'search', 'mapelList'));
+    }
+
+    public function storeGuru(Request $request): RedirectResponse
+    {
+        $validated = $request->validate([
+            'nama' => 'required|string|max:255',
+            'mapel' => 'required|string|exists:mapel,nama_mapel|max:255',
+            'email' => 'required|email|max:255|unique:guru,email',
+        ]);
+
+        Guru::create($validated);
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Guru::create().


(staticMethod.notFound)